reconcile/terraform_resources.py

```diff
@@ -130,15 +131,15 @@
   }
 }
 ... on NamespaceTerraformResourceECR_v1 {
-  account
+  {% if account %}account{% endif %}
   identifier
   region
   output_resource_name
   public
   annotations
 }
 ... on NamespaceTerraformResourceS3CloudFront_v1 {
-  account
+  {% if account %}account{% endif %}
```

Comment:
This is an interesting style. But I see this Jinja-style rendering mixed with `%s` in [other places](https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/queries.py#L425) . Should we agree (and then document) that this is the preferred way when it comes to formatting queries?